src/components/Disputes/DisputeActions.js

```diff
@@ -282,7 +282,7 @@ const useInfoAttributes = ({
 
     // Juror has voted and reveal period hasn't ended
     return {
-      title: 'Your vote was cast successfuly.',
+      title: 'Your vote was cast and revealed successfully.',
```

Comment:
Not sure, but I think at this point the juror might not have revealed yet?

Reply:
Yeah we should show this last part conditionally based on `jurorDraft.outcome`. 
I can push the change, want to check also if the length of the text wouldn't make it look weird. I think is short enough but just in case.

Reply:
Just pushed this last bit.